checkstyle/import-control-core.xml

```diff
@@ -38,6 +38,10 @@
   <allow pkg="kafka.serializer" />
   <allow pkg="org.apache.kafka.common" />
 
+  <!--  use kafka.metrics KafkaYammerMetrics instead -->
```

Comment:
It would be good to include a reference to KIP-544 here so that people knew why to use `KafkaYammerMetrics`.

Reply:
done